kncaputo/fitlit-refactor-tractor
# Dom sad path

## What's this PR do?
- Displays 'N/A' on days that no/partial data exists
- Displays user friends steps in the dropdown
- Fixes bug that did not display trending stairs/steps
- Ensures all methods in scripts do not repeat logic in classes

## Where should the reviewer start?
- `scripts.js` 

## How should this be manually tested?
- Run `npm start` in the terminal and open the localhost site in the browser, change the date and ensure that data is displaying correctly

## Screenshots:
-Sample display on days with no data
<img width="1251" alt="Screen Shot 2020-11-02 at 2 02 25 PM" src="https://user-images.githubusercontent.com/58453566/97918867-12fc2d00-1d14-11eb-8759-f9b669e84b41.png">

- Sample drop down with friends info
<img width="485" alt="Screen Shot 2020-11-02 at 2 02 33 PM" src="https://user-images.githubusercontent.com/58453566/97918897-17c0e100-1d14-11eb-8d5c-8f2ac0ed2a24.png">
